src/Tracy/Panel.php

```diff
@@ -36,12 +33,12 @@ class Panel implements Tracy\IBarPanel
 
 	public function __construct()
 	{
-		if (!class_exists('Tracy\Debugger') && class_exists('Nette\Diagnostics\Debugger')) {
+		if (!class_exists(Debugger::class) && class_exists('Nette\Diagnostics\Debugger')) {
```

Comment:
Tento if-blok je taky jen kompatibilita s Nette 2.1 a je možné jej odstranit. Stačí ponechat to co je v `else` bloku.